ddtrace/contrib/grpc/propagation.py

```diff
@@ -0,0 +1,29 @@
+import grpc
+import collections
+
+class ClientCallDetails(
+        collections.namedtuple(
+            '_ClientCallDetails',
+            ('method', 'timeout', 'metadata', 'credentials')),
+        grpc.ClientCallDetails):
+    """Copy/paste fromhttps://github.com/grpc/grpc/blob/d0cb61eada9d270b9043ec866b55c88617d362be\
```

Comment:
I can't seem to do a suggestion on this/multiple lines, but can we put the link all on a single line so it is copy/pastable?

This line will be really long and probably cause an issue with linting, you can add `  # noqa` (kind of hard to tell in this example, but two spaces, pound, space, "noqa") to the end of the line to disable the linting error